workflow/hosts/gaeac6.yaml

```diff
@@ -1,27 +1,33 @@
-BASE_GIT: '/gpfs/f6/bil-fire8/world-shared/global/glopara/git'
-DMPDIR: '/gpfs/f6/bil-fire8/world-shared/global/glopara/dump'
-BASE_DATA: '/gpfs/f6/bil-fire8/world-shared/global/glopara'
-BASE_IC: '/gpfs/f6/bil-fire8/world-shared/global/glopara/data/ICSDIR'
-PACKAGEROOT: '/gpfs/f6/bil-fire8/world-shared/global/glopara/nwpara'
-COMINsyn: '/gpfs/f6/bil-fire8/world-shared/global/glopara/com/syndat'
-HOMEDIR: '/gpfs/f6/bil-fire8/scratch/${USER}'
-STMP: '/gpfs/f6/bil-fire8/scratch/${USER}'
-PTMP: '/gpfs/f6/bil-fire8/scratch/${USER}'
+BASE_GIT: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/git'
+DMPDIR: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/dump'
+BASE_DATA: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/data'
+BASE_IC: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/data/ICSDIR'
+PACKAGEROOT: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/nwpara'
+COMINsyn: '/gpfs/f6/drsa-precip3/world-shared/role.glopara/com/syndat'
+HOMEDIR: '/gpfs/f6/${ACCOUNT}/scratch/${USER}'
+STMP: '/gpfs/f6/${ACCOUNT}/scratch/${USER}'
+PTMP: '/gpfs/f6/${ACCOUNT}/scratch/${USER}'
 NOSCRUB: $HOMEDIR
-ACCOUNT: bil-fire8
+ACCOUNT: drsa-precip3
```

Comment:
`ACCOUNT` is set after it is being invoked.  I wonder what would happen if line 11 was:
`ACCOUNT: ${ACCOUNT}`
?

Reply:
It's a little confusing, but these fields are referenced as literals when filling in the templated sections of `config.base`.  In `config.base`, `ACCOUNT` is assigned before `HOMEDIR`, `STMP`, etc.  So in `config.base`,
`export ACCOUNT="@ACCOUNT@"`
will become
`export ACCOUNT="drsa-precip3"`
and
`export HOMEDIR="@HOMEDIR"`
will become
`export HOMEDIR="/gpfs/f6/${ACCOUNT}/scratch/${USER}"`

The advantage to this method is that if a user needs to change the default `ACCOUNT` in `config.base` after `setup_expt.py` is called, they only have to change `ACCOUNT` and not `HOMEDIR`, `STMP`, or `PTMP` in their experiments `EXPDIR/config.base`.

I don't think we could assign `ACCOUNT: ${ACCOUNT}` because it isn't a variable that is set when the config files are sourced.